traffic_ops/traffic_ops_golang/config/config.go

```diff
@@ -109,10 +109,12 @@ type ConfigTrafficOpsGolang struct {
 	CRConfigEmulateOldPath bool `json:"crconfig_emulate_old_path"`
 }
 
-// RoutingBlacklist contains the list of route IDs that will be handled by TO-Perl and a list of route IDs that are disabled.
+// RoutingBlacklist contains the list of route IDs that will be handled by TO-Perl, a list of route IDs that are disabled,
+// and whether or not to ignore unknown routes.
 type RoutingBlacklist struct {
-	PerlRoutes     []int `json:"perl_routes"`
-	DisabledRoutes []int `json:"disabled_routes"`
+	IgnoreUnknownRoutes bool  `json:"ignore_unknown_routes"`
```

Comment:
This will default to "False" then, is that what we want? It means existing configs will use that behavior, but idk if that's necessarily a bad thing because they also wouldn't have "PerlRoutes" or "DisabledRoutes".

Reply:
I believe this is what we want. If you intentionally put unknown routes in the config, e.g. in anticipation of disabling routes that don't yet exist, you should also make sure they're explicitly ignored. By default, I'd think you'd want to know if you made a typo.